support-workers/src/main/scala/com/gu/zuora/subscriptionBuilders/ContributionSubscriptionBuilder.scala

```diff
@@ -0,0 +1,29 @@
+package com.gu.zuora.subscriptionBuilders
```

Comment:
these "...Builder" files already existed for some products and in some form, now they are flattened into a consistent set and given simpler and more consistent input/output parameters so that it's easy to jump to the right one and understand what happens in each case.